### ISSUE TITLE
2 - Create streamer service

### ISSUE THREAD

**Issue by carloscasalar** [carloscasalar/youtube-audio-streamer#2]
Create streamer service and use it inside de controller

**Comment by carloscasalar** [comment]
After add some elemental code in aeace52 it has work for a small video like https://youtu.be/g-gp-Voq6MQ but it doesn't with a larger one like https://youtu.be/sNECJax_wq8

* Working test url: http://localhost:3000/stream-by-url?youtubeUrl=https%3A%2F%2Fyoutu.be%2Fg-gp-Voq6MQ
* Non working video url: http://localhost:3000/stream-by-url?youtubeUrl=https%3A%2F%2Fyoutu.be%2FsNECJax_wq8

Error trace:
```
Youtube url: https://youtu.be/sNECJax_wq8
internal/util.js:179
    const signalName = signals[signal.toUpperCase()];
                                      ^

RangeError: Maximum call stack size exceeded
    at convertToValidSignal (internal/util.js:179:39)
    at ChildProcess.kill (internal/child_process.js:381:5)
    at PassThrough.<anonymous> (/opt/app/node_modules/fluent-ffmpeg/lib/processor.js:455:26)
    at emitOne (events.js:115:13)
    at PassThrough.emit (events.js:210:7)
    at writeAfterEnd (_stream_writable.js:222:10)
    at PassThrough.Writable.write (_stream_writable.js:271:5)
    at PassThrough.Writable.end (_stream_writable.js:536:10)
    at emitThree (events.js:140:20)
    at DestroyableTransform.emit (events.js:216:7)
npm info lifecycle youtube-audio-streamer@1.0.0-SNAPSHOT~start: Failed to exec start script
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! youtube-audio-streamer@1.0.0-SNAPSHOT start: `node dist/server.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the youtube-audio-streamer@1.0.0-SNAPSHOT start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-08-07T14_54_36_451Z-debug.log
```

**Comment by carloscasalar** [comment]
Done and merged in PR #11
